packages/instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -969,3 +980,125 @@ describe('pg-pool (ESM)', () => {
     });
   });
 });
+
+describe('pg semantic conventions env variable', () => {
+  let pool: pgPool<pg.Client>;
+  let contextManager: AsyncLocalStorageContextManager;
+  let instrumentation: PgInstrumentation;
+  const provider = new BasicTracerProvider({
+    spanProcessors: [new SimpleSpanProcessor(memoryExporter)],
+  });
+  const testPostgres = process.env.RUN_POSTGRES_TESTS; // For CI: assumes local postgres db is already available
+  const testPostgresLocally = process.env.RUN_POSTGRES_TESTS_LOCAL; // For local: spins up local postgres db via docker
+  const shouldTest = testPostgres || testPostgresLocally; // Skips these tests if false (default)
+  const pgPool = require('pg-pool');
```

Comment:
nit:

```suggestion
  // Here we are `require`ing *before* the instrumentation is created below.
  // In *general* this is a potential instrumentation issue, but this works for
  // `pg-pool` instrumentation because it patches the `pgPool.prototype`
  // (i.e. not a top-level export).
  const pgPool = require('pg-pool');
```